tasks/fix-cat2.yml

```diff
@@ -1423,12 +1562,18 @@
       - RHEL-07-040520
 
 - name: "MEDIUM | RHEL-07-040560 | PATCH | An X Windows display manager must not be installed unless approved."
-  command: "true"
+  yum:
+      name:
+          - "@X Windows System"
```

Comment:
Pretty sure ther is some funny business in the yum module when performing a group uninstall. Might need to enumerate packages and shove them in a variable in `vars/main.yml` if that's cleaner.